boards/arm/npcx7m6fb_evb/doc/index.rst

```diff
@@ -79,10 +79,20 @@ JTAG only sessions.
 Flashing
 ========
 
-Build application as usual for the ``npcx7m6fb_evb`` board, and flash
-using Servo V2, μServo, or Servo V4 (CCD). See the
+If the correct headers are installed, this board supports both J-TAG and also
+the ChromiumOS servo.
+
+To flash using Servo V2, μServo, or Servo V4 (CCD), see the
 `Chromium EC Flashing Documentation`_ for more information.
 
+To flash with J-TAG, install the drivers for your programmer, for example:
+SEGGER J-link's drivers are at https://www.segger.com/downloads/jlink/
+
+Build and flash the blinky sample.::
+
+  west build -t clean && \
+    west build -c -p auto -b npcx7m6fb_evb samples/basic/blinky && \
+    west flash --hex-file build/zephyr/zephyr.npcx.hex
```

Comment:
Ah, want to look if it's possible to have cmake set the right file automatically for npx boards? Mcuboot does it with

https://github.com/zephyrproject-rtos/zephyr/blob/b6165e01caa42a5c2cc3e05744575e11ae142d4b/cmake/mcuboot.cmake#L13-L17

https://github.com/zephyrproject-rtos/zephyr/blob/b6165e01caa42a5c2cc3e05744575e11ae142d4b/cmake/mcuboot.cmake#L142

I think, maybe it can be sneaked in the right spot and this would just be `west flash`

Reply:
I will check it later. Thanks for reminding this.

Reply:
Sorry for replying late. Yes, `west flash` works properly also. I will update it. Thanks.

Reply:
Done.